std/datetime/_constants.ts

```diff
@@ -0,0 +1,6 @@
+// constant times in ms
+export const SECOND = 1e3; // 1000;
+export const MINUTE = 6e4; // SECOND * 60;
+export const HOUR = 36e5; // MINUTE * 60;
+export const DAY = 864e5; // HOUR * 24;
+export const WEEK = 6048e5; // DAY * 7;
```

Comment:
It would be less error prone if you used the commented out expressions

Reply:
Is it useful to have this in a separate module? I'd just put them wherever they are used.